src/android/app/src/main/java/io/github/lime3ds/android/features/settings/model/Settings.kt

```diff
@@ -213,6 +213,7 @@ class Settings {
         const val PREF_THEME_MODE = "ThemeMode"
         const val PREF_BLACK_BACKGROUNDS = "BlackBackgrounds"
         const val PREF_SHOW_HOME_APPS = "ShowHomeApps"
+        const val PREF_DEFAULT_SCREEN_ORIENTATION = "default_device_orientation"
```

Comment:
Why is this string key using a different case than all others?

Reply:
It was initially an `IntSetting()`, they all use this style. It got moved around a lot.

When we have a decision on where to put the setting in the end, i am going to change it to whatever sytle the others in that section uses.